pkg/providers/ovirt/client/ovirt-client.go

```diff
@@ -151,6 +153,44 @@ func (client *richOvirtClient) StopVM(id string) error {
 	}
 }
 
+// StartVM requests VM start and doesn't wait for it to be UP
+func (client *richOvirtClient) StartVM(id string) error {
+	vmService := client.connection.SystemService().VmsService().VmService(id)
+
+	vmResponse, _ := vmService.Get().Send()
+	vm, vmAvailable := vmResponse.Vm()
+	if !vmAvailable {
+		return fmt.Errorf("Failed to start vm %s, vm is not available", id)
+	}
+
+	// Request the VM startup if VM is not UP
+	if status, _ := vm.Status(); status != ovirtsdk.VMSTATUS_UP {
```

Comment:
If VM's state is `powering_up` for example. The call to start VM will fail, which is something we don't need I guess. Thinking of that, I think we need to have also some check/validation for HA VM's because we can't stope those.

Reply:
So in case of HA VMs we cannot perform the migration at all?

Reply:
It seems that starting from down state is the safest approach.

Reply:
Ok, so it seems if it's manual shut-down[0], it's valid:
```
VMs manually shut down or powered off from oVirt, from the guest operating system, or by admin-issued host shutdown (which counts as user initiated shutdown) are not restarted automatically. Switching VMs off manually is a user action and oVirt honors it. The user is responsible for starting such a VM again manually.
```
[0] https://www.ovirt.org/develop/ha-vms.html

Reply:
Yes, for start status down, is OK I think.